homeserver.yaml

```diff
@@ -0,0 +1,37 @@
+# Configuration file for Synapse.
+#
+# This is a YAML file: see [1] for a quick introduction. Note in particular
+# that *indentation is important*: all the elements of a list or dictionary
+# should have the same indentation.
+#
+# [1] https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html
+#
+# For more information on how to configure Synapse, including a complete accounting of
+# each option, go to docs/usage/configuration/config_documentation.md or
+# https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html
+server_name: "my.matrix.host"
```

Comment:
How does this interact with the charm configuration option? Should this perhaps be a template in a `files` directory of the charm?

Reply:
It doesn't, I deleted the file, it creeped into version control. The config lifecycle is via a `generate` command in the container that places a template file and then a `migrate` command that based on the template and environment variables set creates the static `homeserver.yaml` file.